index.html

```diff
@@ -20,19 +22,27 @@
       <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
     <![endif]-->
   </head>
-  <body ng-app="GiphyApp">
+  <body ng-app="GiphyApp" ng-controller="GiphySearch">
     <div class="container">
       <h1 class="text-center">Welcome to Giphy Search</h1>
 
-      <form class="form-inline">
+      <form class="form-inline" ng-submit="searchForGifs()">
         <div class="form-group">
           <label for="search">Search</label>
-          <input type="text" placeholder="Search for GIFs" id="search" class="form-control">
+          <input type="text" placeholder="Search for GIFs" id="search" class="form-control" ng-model="userSearch">
         </div>
         <button type="submit" class="btn btn-primary">Search</button>
       </form>
 
-      <div id="results"></div>
+        <div infinite-scroll="searchMore()" infinite-scroll-distance="0">
+        <div id="results" ng-repeat="gif in gifs track by $index">
+          <img ng-src="{{gif.images.fixed_height.url}}">
+          <!-- {{$index}} -->
+          </div>
+        </div>
+      </div>
+
+
     </div>
```

Comment:
Div element to host the infinite scrolling:

    <div class="row" id="results" infinite-scroll="myPagingFunction()" infinite-scroll-distance="3">
        <div class="well col-md-6" ng-repeat="giphy in giphys track by giphy.id">
            <img ng-src="{{giphy.images.fixed_height.url}}" alt="Description" />
            <a ng-href="{{giphy.images.fixed_height.url}}">link</a>
        </div>
    </div>